features/rageshake/impl/src/main/kotlin/io/element/android/features/rageshake/impl/reporter/DefaultBugReporter.kt

```diff
@@ -82,11 +82,11 @@ class DefaultBugReporter @Inject constructor(
         private const val LOG_CAT_FILENAME = "logcat.log"
         private const val LOG_DIRECTORY_NAME = "logs"
         private const val BUFFER_SIZE = 1024 * 1024 * 50
+        var currentTracingFilter: String? = null
```

Comment:
A bit dirty to put this in the companion, but DefaultBugReporter is not a singleton. I guess it's fine @ganfra ?

Reply:
Why not making it a singleton then?

Reply:
I can do it if you prefer, but there is no need to keep this object in memory, it's only used when the user want to report a problem.